packages/components/notification/src/Notification.tsx

```diff
@@ -7,20 +7,20 @@ import {
   ShowAction,
   CloseAction,
   CloseAllAction,
-  SetPositionAction,
+  SetPlacementAction,
   SetDurationAction,
   NotificationProps,
-  Position,
+  Placement,
```

Comment:
Open question: Do we really want to rename this property? 

In general, I'd avoid doing a breaking change in the component API if it doesn't contradict our guidelines. IMO, in this case, "Position" is an acceptable term.

Reply:
I changed this to keep it like we did for the `Tooltip`, and other components, but I'm fine with either too

Reply:
I prefer we keep the name and prop values the same across them all. If not now, then when?

Reply:
I'm with @denkristoffer on this one

We can make a codemod that rename props, no?

Reply:
alright, let's rename it then